### DISCUSSION TITLE
Could be possible to add property as deform groups?

### DISCUSSION THREAD

**Discussion by sepux** [dtlindsey/bifrost-dl-core#45] [General]
Wondering if this is possible. I playing with your deformers and I was wondering if you could also add support to use properties/attributes as deform groups? 

Thanks.

**Comment by dtlindsey** [comment]
That's essentially what deform groups are, just a mapping of two different mesh properties I've defined.  It'd be good to discuss what kind of property you're looking for, to figure out how it could be built do support data changing under the hood.

**Comment by sepux** [comment]
Hey thanks for having a look at this, I recorded a quick video of what I'm trying to do. I'm sure I'm missing something obvious, let me know if that is the case. 

Thanks

https://user-images.githubusercontent.com/20706047/144958343-f1704232-ab46-4921-8584-8ff8cc4107f8.mp4

**Reply by dtlindsey** [reply]
Ah, I see what you're trying to do.  So you're not missing anything, however, the way I would be addressing this, is with a custom weight volume.

The deform groups are used to isolate what verts are being computed, by isolating the vert table through a list of face elements.  What it looks like you want, is a dynamic weighting of the deformation.  Which would be handled through the weight volumes.  This is on my list of things to do, is to extend the weight volume options, currently you only have a sphere and a plane, but it could just as easily be setup to take a vertex parameter to assign the weighting.  If you're on the Bifrost discord, I'm happy to discuss via DM?

**Comment by dtlindsey** [comment]
https://github.com/dtlindsey/bifrost-dl-core/issues/47#issue-1073014420
